android/app/build.gradle

```
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion flutter.compileSdkVersion
+    compileSdkVersion 33
```

```suggestion
    compileSdkVersion flutter.compileSdkVersion
```
Not sure if we want to change this to a specific version.

---

> 

I was getting an error using flutter.compileSdkVersion, but it seems to compile anyways. Changing it back to that now.